docs/log_collection.md

```diff
@@ -90,6 +90,42 @@ Send logs to Datadog from your Android applications with [Datadog's `dd-sdk-andr
 ```
 {{% /tab %}}
 {{< /tabs >}}
+{{< /site-region >}}
+
+{{< site-region region="gov" >}}
+{{< tabs >}}
+{{% tab "Kotlin" %}}
+```kotlin
+    class SampleApplication : Application() {
+        override fun onCreate() {
+        super.onCreate()
+        val configuration = Configuration.Builder(logsEnabled = true, ...)
+                            .useSite(DatadogSite.US1_FED)
+                            .build()
+        val credentials = Credentials(<CLIENT_TOKEN>, <ENV_NAME>, <APP_VARIANT_NAME>, <APPLICATION_ID>)
+        Datadog.initialize(this, credentials, configuration, trackingConsent)
+        }
+    }
+```
+{{% /tab %}}
+{{% tab "Java" %}}
+```java
+    public class SampleApplication extends Application {
+        @Override
+        public void onCreate() {
+            super.onCreate();
+            Configuration configuration =
+                    new Configuration.Builder(true, true, true, true)
+                            .useSite(DatadogSite.US1_FED)
+                            .build();
+            Credentials credentials = new Credentials( < CLIENT_TOKEN >, <ENV_NAME >, <APP_VARIANT_NAME >, <
+            APPLICATION_ID >);
+            Datadog.initialize(this, credentials, configuration, trackingConsent);
+        }
+    }
+```
+{{% /tab %}}
+{{< /tabs >}}
```

Comment:
If I understand correctly, we'd need to duplicate this for all region : 
- `us1` -> no need to specify `useSite`
- `us3` -> `useSite(DatadogSite.US3)`
- `us5` -> `useSite(DatadogSite.US5)`

Reply:
Hey @xgouchet yes, that's correct. Good catch! I've added US5. 

Right now we have a note saying it's not supported for US3: https://github.com/DataDog/dd-sdk-android/pull/786/files#diff-f8009728ad3cada081700714b0842087e7cd544d58d4ae7d8b38baafb090c717R4

Could you confirm that this works for US3? Because Sarina mentioned: https://github.com/DataDog/dd-sdk-android/pull/786#discussion_r768999311